strands_human_aware_navigation/scripts/human_aware_navigation.py

```diff
@@ -116,7 +110,8 @@ def resetSpeed(self):
 
     def pedestrianCallback(self, pl):
         if not self._as.is_active():
-            rospy.logdebug("No active goal")
+            rospy.logdebug("No active goal. Unsubscribing.")
+            self.ppl_sub = None
```

Comment:
`Subscriber.unregister()` actually seems to cause some problems as we found out during the winter school and other people are struggling with on anwers.ros.org, e.g. [here](http://answers.ros.org/question/203735/unsubscribing-from-topics/)
Setting it to `None` lets the subscriber go out of scope and it therefore triggers the unsubscribing automatically.

Still have to test on the robot once the experiments are done.